content/docs/operating/security.md

```diff
@@ -12,12 +12,13 @@ environments.
 This page describes the general security assumptions of Prometheus and the
 attack vectors that some configurations may enable.
 
-As with any complex system, it is not possible to guarantee that there are no
-bugs. If you find a security bug please report it privately to the maintainers
-listed in the MAINTAINERS.md of the relevant repository and CC
-prometheus-team@googlegroups.com. We will fix the issue and coordinate a
-release date with you, acknowledging your effort and mentioning you by name
-if you want.
+As with any complex system, it is certain that bugs will be found, some of
```

Comment:
It's not certain, just extremely likely.

Reply:
Without descending into the halting problem and the differences of having and finding: Do you care for this to be changed or can it be merged as-is?

Reply:
I care, as the change as proposed is incorrect.

Reply:
Is there a misunderstanding here regarding "it is certain that bugs will be found"? Because the first time I read it, I parsed it as "any bugs that exist will be found", but then I realized what's meant is probably that people will find bugs, but obviously not all of them (which makes much more sense).

Reply:
My reading of it is that it is 100% certain that bugs will be found (the exact sense of that doesn't matter) - and you can't ever be 100% certain of things, particularly into the future.